### PR TITLE
[docs][explore] Formatting fixes

### PR DESCRIPTION
**Pull request by ddhodge**
Fixing formatting in Indexes section

@netlify /preview/explore/indexes-constraints/

### CODE REVIEW

File: docs/content/preview/explore/indexes-constraints/partial-index-ysql.md [hunk 1]
```diff
@@ -54,35 +57,35 @@ SELECT * FROM customers LIMIT 3;
 ```output
 customer_id |       company_name        |  contact_name  |    contact_title    |           address           |   city    | region | postal_code | country |     phone      |      fax
```
Comment (polarweasel) [inline docs/content/preview/explore/indexes-constraints/partial-index-ysql.md:58]: SQL output tables are weird. Need leading spaces on this and the ones you removed
```suggestion
 customer_id |       company_name        |  contact_name  |    contact_title    |           address           |   city    | region | postal_code | country |     phone      |      fax
```

File: docs/content/preview/explore/indexes-constraints/partial-index-ysql.md [hunk 2]
```diff
@@ -54,35 +57,35 @@ SELECT * FROM customers LIMIT 3;
...
- FAMIA       | Familia Arquibaldo        | Aria Cruz      | Marketing Assistant | Rua Orós, 92                | Sao Paulo | SP     | 05442-030   | Brazil  | (11) 555-9857  |
- VINET       | Vins et alcools Chevalier | Paul Henriot   | Accounting Manager  | 59 rue de l'Abbaye          | Reims     |        | 51100       | France  | 26.47.15.10    | 26.47.15.11
- GOURL       | Gourmet Lanchonetes       | André Fonseca  | Sales Associate     | Av. Brasil, 442             | Campinas  | SP     | 04876-786   | Brazil  | (11) 555-9482  |
+FAMIA       | Familia Arquibaldo        | Aria Cruz      | Marketing Assistant | Rua Orós, 92                | Sao Paulo | SP     | 05442-030   | Brazil  | (11) 555-9857  |
+VINET       | Vins et alcools Chevalier | Paul Henriot   | Accounting Manager  | 59 rue de l'Abbaye          | Reims     |        | 51100       | France  | 26.47.15.10    | 26.47.15.11
+GOURL       | Gourmet Lanchonetes       | André Fonseca  | Sales Associate     | Av. Brasil, 442             | Campinas  | SP     | 04876-786   | Brazil  | (11) 555-9482  |
```
Comment (polarweasel) [inline docs/content/preview/explore/indexes-constraints/partial-index-ysql.md:62]:
```suggestion
 FAMIA       | Familia Arquibaldo        | Aria Cruz      | Marketing Assistant | Rua Orós, 92                | Sao Paulo | SP     | 05442-030   | Brazil  | (11) 555-9857  |
 VINET       | Vins et alcools Chevalier | Paul Henriot   | Accounting Manager  | 59 rue de l'Abbaye          | Reims     |        | 51100       | France  | 26.47.15.10    | 26.47.15.11
 GOURL       | Gourmet Lanchonetes       | André Fonseca  | Sales Associate     | Av. Brasil, 442             | Campinas  | SP     | 04876-786   | Brazil  | (11) 555-9482  |
```

File: docs/content/preview/explore/indexes-constraints/partial-index-ysql.md [hunk 3]
```diff
@@ -54,35 +57,35 @@ SELECT * FROM customers LIMIT 3;
...
 (3 rows)
 ```
 
-- Let's say you want to query the subset of customers who are `Sales Managers` in the `USA`. The query plan using the `EXPLAIN` statement would look like the following:
+Suppose you want to query the subset of customers who are `Sales Managers` in the `USA`. The query plan using the `EXPLAIN` statement would look like the following:
```
Comment (polarweasel) [inline docs/content/preview/explore/indexes-constraints/partial-index-ysql.md:66]: We could live without the `code font` on Sales Managers and USA, for sure... We should try to kill this sort of usage wherever we find it

File: docs/content/preview/explore/indexes-constraints/partial-index-ysql.md [hunk 4]
```diff
@@ -91,16 +94,13 @@ northwind=# EXPLAIN SELECT * FROM customers where (country = 'USA' and contact_t
...
-
 ```
 
 ## Learn more
 
 - [SQL Puzzle: Partial Versus Expression Indexes](https://blog.yugabyte.com/sql-puzzle-partial-versus-expression-indexes/)
-
 - [The Benefit of Partial Indexes in Distributed SQL Databases](https://blog.yugabyte.com/the-benefit-of-partial-indexes-in-distributed-sql-databases/)
-
 - [Indexes on JSON attributes](/preview/explore/json-support/jsonb-ysql/#6-indexes-on-json-attributes)
```
Comment (polarweasel) [inline docs/content/preview/explore/indexes-constraints/partial-index-ysql.md:106]: Relative references, please...

File: docs/content/preview/explore/indexes-constraints/foreign-key-ysql.md [hunk 5]
```diff
@@ -54,9 +54,7 @@ Defining the `CONSTRAINT` clause and naming the foreign key is optional. If you
 
 ## Examples
 
-- To run the examples below, follow these steps to create a local [cluster](/preview/quick-start/) or in [YugabyteDB Managed](/preview/yugabyte-cloud/cloud-connect/).
-
-- Use the [YSQL shell](/preview/admin/ysqlsh/) for local clusters, or [Connect using cloud shell](/preview/yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed, to create a database.
+Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ycqlsh](../../../admin/ycqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```
Comment (aishwarya24) [inline docs/content/preview/explore/indexes-constraints/foreign-key-ysql.md:57]:
```suggestion
Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ysqlsh](../../../admin/ysqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```

File: docs/content/preview/explore/indexes-constraints/primary-key-ycql.md [hunk 6]
```diff
@@ -41,11 +41,7 @@ Refer to the Grammar section for [CREATE TABLE](../../../api/ycql/ddl_create_tab
 
 ## Examples
 
-To run the examples, do the following:
-
-- For local clusters, follow the steps in [Quick Start](../../../quick-start/) to install YugabyteDB, and use the [YCQL shell](../../../admin/ycqlsh/).
-
-- For YugabyteDB Managed, create a cluster and [connect using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/).
+Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ysqlsh](../../../admin/ysqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```
Comment (aishwarya24) [inline docs/content/preview/explore/indexes-constraints/primary-key-ycql.md:44]:
```suggestion
Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ycqlsh](../../../admin/ycqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```

File: docs/content/preview/explore/indexes-constraints/unique-index-ycql.md [hunk 7]
```diff
@@ -33,60 +33,61 @@ type: docs
...
 
 ```sql
-CREATE INDEX index_name ON table_name(column_list);
+CREATE UNIQUE INDEX index_name ON table_name(column_list);
 ```
 
 ## Example
 
-- Follow the steps to create a cluster [locally](/preview/quick-start/) or in [YugabyteDB Managed](/preview/yugabyte-cloud/cloud-connect/).
+Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ysqlsh](../../../admin/ysqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```
Comment (aishwarya24) [inline docs/content/preview/explore/indexes-constraints/unique-index-ycql.md:48]:
```suggestion
Create a cluster [locally](../../../quick-start/) or in [YugabyteDB Managed](../../../yugabyte-cloud/cloud-basics/create-clusters-free/) and connect to the cluster using [ycqlsh](../../../admin/ycqlsh/) for local clusters, or [using cloud shell](../../../yugabyte-cloud/cloud-connect/connect-cloud-shell/) for YugabyteDB Managed.
```

**Review by aishwarya24 — APPROVED** [review]

**Pull request merged**